cim-forward-route/src/main/java/com/crossoverjie/cim/route/service/impl/AccountServiceRedisImpl.java

```diff
@@ -55,6 +55,9 @@ public class AccountServiceRedisImpl implements AccountService {
     @Autowired
     private OkHttpClient okHttpClient;
 
+    @Autowired
```

Comment:
```suggestion
    @Resource
```

We will use `@Resource` uniformly, and the existing `@Autowired` also needs to be replaced.